SWF-005-T1/SWF-005-T1.py

```diff
@@ -15,50 +15,94 @@
 from astropy import coordinates
 from astropy import units as u
 import matplotlib.pyplot as plt
+import logging
+import yaml
+import os
 
+logger = logging.getLogger(__name__)
 
-# Configuration
-# Central Right Ascension coordinate for query
-RA = coordinates.Angle("10h50m6.6s")
-# Central Declination coordinate for query
-DEC = coordinates.Angle("30d40m35s")
-# Coordinate system frame
-FRAME = "icrs"
+def main():
+    
+    # Path to the directory containing the required visibility files:
+    # - eor.uvfits
+    # - foregrounds.uvfits
+    config_file = './config/config.yml'
+    if os.path.exists(config_file):
+        with open(config_file, "r") as f:
+            config = yaml.safe_load(f)
+    else:
+        warnings.warn(f"! Configuration file '{config_file}' not found. Using default settings.", UserWarning)
+        config = {}
 
+    default_result_path = "../results"
+    result_path = config.get("result_path")
+    if result_path is None:
+        warnings.warn(f"! 'result_path' not found in '{config_file}'. Using default: '{default_result_path}'.", UserWarning)
+        result_path = default_result_path
+    result_path += "/SWF-005-T1/"
```

Comment:
I think it is more robust to use `os.path.join` rather than treating these paths as strings. This fails for me whenever there are missing backslashes etc
```
new_path = os.path.join(results_path, 'SWF-005-T2')
```